test/commands/version.test.ts

```diff
@@ -52,12 +52,13 @@ describe('version', () => {
   .stdout()
   .command(['version', '--json', '--verbose'])
   .end('runs version --verbose --json', output => {
-    expect(JSON.parse(output.stdout)).to.deep.equal({
-      architecture: `${process.platform}-${process.arch}`,
-      cliVersion: `@oclif/plugin-version/${pjson.version}`,
-      nodeVersion: `node-${process.version}`,
-      osVersion: `${osType()} ${osRelease()}`,
-      pluginVersions: [],
-    })
+    const json = JSON.parse(output.stdout)
+    expect(json).to.have.property('architecture', `${process.platform}-${process.arch}`)
+    expect(json).to.have.property('cliVersion', `@oclif/plugin-version/${pjson.version}`)
+    expect(json).to.have.property('nodeVersion', `node-${process.version}`)
+    expect(json).to.have.property('osVersion', `${osType()} ${osRelease()}`)
+    expect(json).to.have.property('pluginVersions')
+    expect(json).to.have.property('shell', getShell())
+    expect(json).to.have.property('rootPath', process.cwd())
```

Comment:
What is the expectation for `pluginVersions`?  It seems to have changed without anything from this PR.  I only changed it from deep equal because the test code was starting to re-implement the runtime code.

Reply:
Ah, I see what you mean. Yeah, that's weird. Not sure what would have changed since plugin-help is just listed as a dev plugin but this is a fine workaround.